scripts/benchmark/benchmark-build-speed

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -eux
+
+if ! command -v gradle-profiler &> /dev/null
+then
+    echo "Make sure to install gradle-profiler - brew install gradle-profiler"
+    exit
+fi
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd $SCRIPT_DIR
+
+git clone git@github.com:duckduckgo/Android.git
+
+cd Android
+git submodule update --init --recursive
+cd ..
+
+gradle-profiler --benchmark \
+ --project-dir Android \
+ --scenario-file duckduckgo/duckduckgo.scenarios \
+ --output-dir results/pre-sentry/
+
+git apply duckduckgo/add-sentry-to-duckduckgo.patch
+
+gradle-profiler --benchmark \
```

Comment:
Is it possible the second run relies on a "warm" gradle service that will skew the results? If so, does it make sense to have a "warm up phase" before the first run to get the service starting?

Reply:
nope, if you take a look at [scenarios](https://github.com/getsentry/sentry-android-gradle-plugin/blob/feat/benchmark-script/scripts/benchmark/duckduckgo/duckduckgo.scenarios) they allow us to specify number of warm-ups and whether it should run with a warm or cold daemon. So this means every execution is being started from a clean state. However, I added `rm -rf gradle-user-home` just to make sure.